src/vm/eeconfig.cpp

```diff
@@ -393,6 +405,9 @@ HRESULT EEConfig::Init()
     // statically link to EEConfig.
     CLRConfig::RegisterGetConfigValueCallback(&GetConfigValueCallback);
 
+#ifndef FEATURE_PAL
+    SetConsoleCtrlHandler(CtrlHandler, TRUE);
```

Comment:
It would be nice if this is close to `PAL_SetTerminationRequestHandler` where the equivalent thing is done for Unix.